students/BrianNisonger/Lesson3/assignment/test_customer_db.py

```diff
@@ -0,0 +1,169 @@
+from basic_operations import add_customer
+from basic_operations import search_customer
+from basic_operations import delete_customer
+from basic_operations import update_customer_credit
+from basic_operations import list_active_customers
+
+from customer_model import Customer
+import pytest
+
```

Comment:
Nice fixtures. Pytest has fixtures you can use to make life easier. https://docs.pytest.org/en/latest/fixture.html